ballerina-release-notes-0.990.0.md

```diff
@@ -384,11 +448,21 @@ public function invokeWorkers() returns int {
 ```
 
 ## Variable Initialization
+### Compile time constants
+A compile-time constant is a typed identifier whose value is computed at compile time. Compile time reference values are frozen values 
+```ballerina
+const i = 10;
+```
+
+### Final variables
+The final modified can be applied to a variable declaration to mean that the variable cannot be modified after it has been initialized
```

Comment:
Shall we say "Modified final variable" or something similar? The "final modified" has incomplete meaning for a newer user.